prow/config.yaml

```diff
@@ -15648,14 +15648,14 @@ periodics:
   spec:
     containers:
     - args:
+      - --bare
```

Comment:
Works locally.

Other e2e jobs use `--bare` here, bootstrap.py will find the scenario and run from there, as no additional info is needed. (presubmit ones still need to checkout the PR)